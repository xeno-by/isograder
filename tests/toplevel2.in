-1: blah
* Section [/2]