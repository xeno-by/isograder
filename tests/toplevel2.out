EXIT CODE:
1

STDOUT:

STDERR:
error parsing the correction sheet:
line 1: at the top level of a correction sheet, only blank lines and section lines are allowed
-1: blah
^

